openedx_learning/apps/authoring/publishing/model_mixins/publishable_entity.py

```diff
@@ -5,12 +5,14 @@
 
```

Comment:
Should this just get folded back as `model_mixins.py`, since there we don't have any container mixins any longer?

Reply:
Yeah, might as well keep it where it was. Done.